plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py

```diff
@@ -29,26 +29,77 @@
 import requests
 
 
+DEFAULT_IPFS_URL = "/ip4/127.0.0.1/tcp/5001"
+ALLOWED_CONNECTION_TYPES = ("tcp",)
+ALLOWED_ADDR_TYPES = ("ip4", "dns")
```

Comment:
on line 36 we also list ip6, so use that too here?

Reply:
Line 36 is just a standard address format we don't necessarily need to follow that. Also, I have allowed only ip4 and dns because they are easy to set up and use. We don't want to make it complicated, at least for now.